lighthouse-core/test/fraggle-rock/api-test-pptr.js

```diff
@@ -194,7 +192,6 @@ describe('Fraggle Rock API', () => {
 
       const {lhr} = result;
       const {auditResults, failedAudits, erroredAudits} = getAuditsBreakdown(lhr);
-      // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
```

Comment:
I'm kinda 50/50 on these. The comment does say "compatibility"

Reply:
Of all the ones left, I'm fine leaving these in :)